Gruntfile.js

```diff
@@ -169,7 +170,7 @@ module.exports = function(grunt) {
             },
             scss: {
                 files: ['src/**/*.scss'],
-                tasks: ['sass', 'copy:styles', 'embed-styles:code', 'drop:dev'],
+                tasks: ['exec:generate-scss-typings', 'sass', 'copy:styles', 'embed-styles:code', 'drop:dev'],
```

Comment:
Isn't this redundant with `yarn watch:scss`?

Reply:
Correct! Removed!